TechForPR/chefsForPR
# Fixes #8 Addded bootstrap CSS and JS to the project

# Solves issue Number: #8 

## Description

Adds the CSS and JS from bootstrap, to provide a baseline for the appeareance of the site.

## Steps to Test or Reproduce

Navigate to the app and see that bootstrap is present.

We're using v3, https://getbootstrap.com/docs/3.3/ 

Consult the documentation to see what components are available.

And using the theme cosmo, https://bootswatch.com/cosmo/
